lms/djangoapps/grades/models.py

```diff
@@ -193,12 +193,12 @@ def cached_get_or_create(cls, user_id, blocks):
                 # another user may have had this block hash created,
                 # even if the user we checked the cache for hasn't yet.
                 model, _ = cls.objects.get_or_create(
-                    hashed=blocks.hash_value, blocks_json=blocks.json_value, course_id=blocks.course_key,
+                    hashed=blocks.hash_value.decode('utf-8'), blocks_json=blocks.json_value, course_id=blocks.course_key,
```

Comment:
b'' prefix is going with hashed value in db. I just decode it and save without it in db.